CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(spatio_temporal_voxel_layer)
-add_compile_options(-std=c++17)
+
+add_definitions (-std=c++14)
```

Comment:
Can we bring this back to 17?

Reply:
We can, but I don't see the reason since there aren't any c++17 functions in use right now . Anyway I'll change it if you think it's better.

Reply:
Please do change back, sorry its just a future proofing thing